samples/test-app/README.md

```diff
@@ -0,0 +1,41 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+We will assume you are in the directory `samples\test-app` while following the further instructions 
+
+## Installation
+
+Run the following commands from your shell.
+
+### Installing the test-app
+
+Inside `test-app`, run
+```bash
+npm install
```

Comment:
This should also have `npm build` command for the SDK.

Reply:
Done